modules/engine/src/index.ts

```diff
@@ -1528,6 +1528,11 @@ export class VectorEngine implements IVectorEngine {
     return Result.ok(results);
   }
 
+  private async syncDisputes(): Promise<Result<void, EngineError>> {
+    await this.vector.syncDisputes();
+    return Result.ok(undefined);
+  }
```

Comment:
This should be wrapped in a  `try-catch` block or you'll get `UnhandledPromiseRejection` errors